blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -59,25 +56,29 @@ class OrganizationPipelines extends Component {
                     // crud operations are relative low frequency, so not much
                     // benefit to be got from optimizing things here.
                     // TODO: fix https://issues.jenkins-ci.org/browse/JENKINS-35153 for delete
-                    _this.props.fetchPipelines(_this.context.config, _this.props.organization);
-                    _this.props.updateBranchList(eventCopy, _this.context.config);
+                    if (this._getOrganizationName()) {
```

Comment:
similar pattern again. Isn't this already assigned a variable above? is `this` something else now?